cookbook/docs/ml_training.rst

```diff
@@ -29,6 +39,39 @@ ML Training
 
     auto/case_studies/ml_training/pima_diabetes/index
     auto/case_studies/ml_training/house_price_prediction/index
+    auto/case_studies/ml_training/pytorch/index
 
 
 .. TODO: write tutorials for data parallel training, distributed training, and single node training
+
+Run the following ``flytectl`` commands to execute your code (replace ``<code_directory>`` with the directory containing your code):
```

Comment:
aren't these instructions similar to https://docs.flyte.org/projects/cookbook/en/latest/index.html#setting-up-your-environment-to-run-the-examples?

Why not point to those instructions there instead of duplicating them here?

Reply:
Sure.